services/app-api/prisma/schema.prisma

```diff
@@ -43,18 +43,33 @@ model State {
 }
 
 model User {
-  id                 String             @id
-  createdAt     DateTime               @default(now())
-  updatedAt     DateTime               @default(now()) @updatedAt
-  givenName          String
-  familyName         String
-  email              String
-  role               Role
-  divisionAssignment Division?
-  stateAssignments   State[]
-  stateCode          String?
-  questions          Question[]
-  responses          QuestionResponse[]
+  id                    String             @id
+  createdAt             DateTime           @default(now())
+  updatedAt             DateTime           @default(now()) @updatedAt
+  givenName             String
+  familyName            String
+  email                 String
+  role                  Role
+  divisionAssignment    Division?
+  stateAssignments      State[]
+  stateCode             String?
+  questions             Question[]
+  responses             QuestionResponse[]
+  audits                UserAudit[]
+  updatedByAuditEntries UserAudit[]        @relation("UpdatedBy")
+}
+
+model UserAudit {
+  id              String      @id @default(uuid())
+  createdAt       DateTime    @default(now())
+  updatedAt       DateTime    @default(now()) @updatedAt
+  modifiedUserId  String      @map("userId")
```

Comment:
I made this comment on the past PR and mistakenly strike it out.

> Do we still need `@default(now())` for `updatedAt`? Since this is a new table, we would not have old records without this field. Or am I recalling how this works completely wrong

So this question still stands.

Reply:
Honestly, I'm not sure we even need an updatedAt column, because each row will be written and never changed.  But I didn't want to break any auto-checkers that we introduce in the future--since we discussed always having createdAt and updatedAt in each table--so I left it in.  But I think you're right that we don't need to specify the default.  I'll take it out.